bundles/InstallBundle/Command/InstallCommand.php

```diff
@@ -115,6 +115,12 @@ private function getOptions()
                 'default' => 3306,
                 'group' => 'db_credentials',
             ],
+            'is-myisam-supported' => [
+                'description' => 'If set to "no" then MyISAM engine is changed to InnoDB in all tables (useful e.g. in Azure database for MySQL)',
+                'mode' => InputOption::VALUE_REQUIRED,
+                'default' => 'yes',
```

Comment:
I think we should use a bool here, and call the option `force-innodb-tables`.
Then if the option is set we set the flag accordingly. I think we should then do the same with MEMORY tables. 
Thanks!

Reply:
To be true I'm not sure about it - MyISAM problem is that it has "lack of transaction support which can lead to cases of data loss." - that's why e.g Azure doesn't support it (https://feedback.azure.com/forums/597982-azure-database-for-mysql/suggestions/19271050-add-myisam-engine-support)

However each engine has got its own advantages (probably that's why not all tables are by default InnoDb). So in my opinion we shouldn't have one potion to force all tables to be the same but according to use cases we should have possibility to "turn off" this special functionalities. And in the Azure database use-case we would like to "turn off" only the MyISAM, becasue why we should also abandon MEMORY engine advantages?

@brusch what do you think?

Reply:
@mgomse The MEMORY storage engine also doesn't support transactions 😉
See: https://mariadb.com/kb/en/library/show-engines/

Reply:
But we're going away from the MEMORY engine anyway, see: #4901

Reply:
But please, let's rename the flag and make it a boolean, then we can merge it. Thanks!